Runtimes/Unity/BepInEx.Unity.Mono/Bootstrap/UnityChainloader.cs

```diff
@@ -23,6 +24,14 @@ namespace BepInEx.Unity.Mono.Bootstrap;
 /// </summary>
 public class UnityChainloader : BaseChainloader<BaseUnityPlugin>
 {
+    internal static readonly ConfigEntry<bool> ConfigHideBepInExGOs = ConfigFile.CoreConfig.Bind(
+     "Chainloader", "HideManagerGameObject",
+     true, new StringBuilder()
+            .AppendLine("If enabled, hides BepInEx Manager GameObject from Unity.")
+            .AppendLine("This can fix loading issues in some games that attempt to prevent BepInEx from being loaded.")
+            .AppendLine("Use this only if you know what this option means, as it can affect functionality of some older plugins.")
```

Comment:
```suggestion
            .AppendLine("This fixes loading issues in some games that attempt to prevent BepInEx from being loaded or have unusual init.")
            .AppendLine("Turn this off only if you know what this option means. Some very old BepInEx5 plugins may need this to be off.")
```